.gitignore

```diff
@@ -5,8 +5,8 @@ mstpd
 
 /bridge-stp
 /bridge-stp.tmp
-/lib/ifupdown.sh
```

Comment:
this rename of `lib` ==> `utils` i'd drop
it's a preference change
it could be that we'd later get a preference change [from someone else] to rename back, or to some other folder name

so, rather than going down that path, i'd prefer to drop this